projects/storefrontlib/src/layout/breakpoint/breakpoint.service.spec.ts

```diff
@@ -123,39 +141,44 @@ describe('BreakpointService', () => {
     });
 
     it('should return xs for <= 576', () => {
-      MockWindow.target.innerWidth = 576;
+      const INNER_WIDTH = 576;
```

Comment:
yeah, I don't think it's so magic, we can revert that too

Reply:
reverted